### DISCUSSION TITLE
Index Table Column Titles Missing

### DISCUSSION THREAD

**Discussion by GlenWarren** [orchidsoftware/platform#2576] [Q&A] [answered]
We have upgraded our packages from:
```
      "orchid/platform": "12.2.1",
      "laravel/fortify": "^1.11",
      "orchid/fortify": "^0.0.8"
```
to:
```
      "orchid/platform": "13.10.0",
      "laravel/fortify": "1.16.0",
      "orchid/fortify": "0.0.11"
```

Following this we are experiencing something strange.

We have several "index" tables that by default are configured to return 0 records. We only display results when a filter has been applied, using the filters on the column titles (so only when an admin is searching for something specific).

After updating the packages, the columns titles and filters are no longer visible when landing on the screen:
<img width="933" alt="Screenshot 2023-04-01 at 14 27 14" src="https://user-images.githubusercontent.com/44340845/229291882-acc1be90-7130-4a9b-bc52-4d067b08b93a.png">

The missing filters and titles prevents admins from being able to search for records.

Does anyone have any idea why this is happening now?

**Comment by tabuna** [comment, marked as answer]
When there are no filter options and no data, it outputs a headerless block for a cleaner interface. You can override this method. For example

https://github.com/orchidsoftware/platform/blob/f9f80c675448f5b8d6a63946b08d5944ca14666d/src/Screen/Layouts/Table.php#L158-L165

**Reply by GlenWarren** [reply]
Thanks, overriding this function has fixed the issue.
